src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.helper.Storage;
+import duke.helper.Ui;
+import duke.task.TaskList;
+
+/**
+ * Base class for classes that execute logic based on user commands.
+ */
+public abstract class Command {
+    protected String filePath;
+    protected String[] inputSplit;
```

Comment:
Consider creating a unique parser for each command, as each command has different number of parameters / input